cli/src/identity.rs

```diff
@@ -0,0 +1,29 @@
+use std::mem::replace;
+
+use openmls::prelude::*;
+
+pub struct Identity {
+    pub(crate) kpb: KeyPackageBundle,
+    pub(crate) credential: CredentialBundle,
+}
+
+impl Identity {
+    pub(crate) fn new(ciphersuite: CiphersuiteName, id: &[u8]) -> Self {
+        let credential_bundle =
+            CredentialBundle::new(id.to_vec(), CredentialType::Basic, ciphersuite).unwrap();
+        let key_package_bundle =
```

Comment:
only one key package bundle?

Reply:
Yeah, since we only take one key package bundle from another client there's no point in creating more 😉 
But this certainly needs more logic to handle key packages properly (like deleting on the server again etc.). Currently essentially there's only a last resort key package right now...